PyDSS/modes/QSTS.py

```diff
@@ -34,8 +31,11 @@ def __init__(self, dssInstance, SimulationSettings, Logger):
         self._dssIntance = dssInstance
         self._dssSolution = dssInstance.Solution
         self._dssSolution.Mode(2)
+
         self._dssSolution.Hour((StartDay - 1) * 24)
-        self._dssSolution.Seconds(StartTimeMin * 60)
+        self._dssSolution.Seconds((StartTimeMin + 0.50000001) * 60.0)
```

Comment:
The reason for this is extremely non-obvious. Can you add a comment explaining why?

Reply:
I believe this works for cases where profile resolution is smaller than the time step. Removing it in any case as it does not work for all cases

Reply:
I have done some code cleanup here. For DISCO you will be implementing a new class right?